tests/unit/models/command-test.js

```diff
@@ -568,13 +572,16 @@ describe('models/command.js', function() {
         td.when(command._printCommand(), {ignoreExtraArgs: true}).thenReturn(' command printed');
       });
 
+      afterEach(function() {
+        td.reset();
+      });
+
       it('calls printCommand', function() {
```

Comment:
Similarly we should probably rename this to something like `"prints the expected help string"` or something.